transports/p2p/server.go

```diff
@@ -509,7 +509,7 @@ func (sp *serverPeer) OnAddr(_ *peer.Peer, msg *wire.MsgAddr) {
 
 		// Add address to known addresses for this peer.
 		sp.addKnownAddresses([]*wire.NetAddress{na})
-		sp.log.Debug().Msgf("Added address %s from host %s", na.IP.String(), sp.NA().IP.String())
+		sp.log.Trace().Msgf("Added address %s from host %s", na.IP.String(), sp.NA().IP.String())
```

Comment:
We're now receiving more then 600 addresses from one seed-peer and we're connecting to 8 of them. So this log is pretty long.